hangman/game.py

```diff
@@ -1,13 +1,131 @@
 from .exceptions import *
-
+import random
 
 class GuessAttempt(object):
-    pass
-
+    
+    def __init__(self, character, hit=False, miss=False):
+        self.character = character
+        self.miss = miss
+        self.hit = hit
+        
+        if hit and miss:
+            raise InvalidGuessAttempt
+            
+        
+    def is_hit(self):
+        if self.hit:
+            return True
+        
+        if not self.hit: 
+            return False
```

Comment:
seems like `self.hit` is already in Boolean data type, you can simply `return self.hit`